README.md

```diff
@@ -1 +1,22 @@
-# microverse-ltr
\ No newline at end of file
+# microverse-ltr
+
+# Built With
+
+    -HTML
+    -CSS
+    -GIITHUB
+
+
+## Authors
+
+**Author1**
+
+- GitHub: [@githubhandle](https://github.com/danielsafari143)
+- Twitter: [@twitterhandle](https://twitter.com/DanielSafari143)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/daniel-safari-a01744251/)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub]
```

Comment:
[OPTIONAL] 👍🏽 

- Hi @danielsafari143, thank you for implementing all the required changes. But I'd love to kindly suggest that for future projects, you stick with the microverse approved template for your  README.

Here is a link to the  [Microverse approved template](https://github.com/microverseinc/readme-template/blob/master/README.md)

I believe this will make your readme look more professional 😃 🍷